sdk/vision/Azure.AI.Vision.ImageAnalysis/CHANGELOG.md

```diff
@@ -5,7 +5,9 @@
 ### Features Added
 
```

Comment:
Replace `Unreleased` in `1.0.0-beta.2 (Unreleased)` with a release date. I used `2024-02-09` for Java & Python, being overly optimistic, but I guess you can put anything.
Remove empty section titles: Features Added, Bugs Fixed, Other Changes